scrapy/signalmanager.py

```diff
@@ -1,11 +1,10 @@
 from __future__ import absolute_import
-from pydispatch import dispatcher
 from scrapy.utils import signal as _signal
```

Comment:
I wonder if we should drop this, directly call the `Signal` methods, and put deprecation methods into the util functions? They are pass-throughs now anyway.